.circleci/config.yml

```diff
@@ -47,6 +47,20 @@ jobs:
       - run: make V=1 -j32 release | .circleci/cat_ignore_eagain
       - run: ./db_stress --version # ensure with gflags
 
+  build-linux-release-rtti:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: 2xlarge
```

Comment:
should we use `large` host to save cost?

Reply:
Good idea. Let me change to that.